### PR TITLE
Update .gitignore

### PR DESCRIPTION
**Pull request by davidgraeff**
* Remove superfluous marytts ignores
* Add .settings/org.eclipse.*: They are auto-generated and always the same

Signed-off-by: David Gräff <david.graeff@web.de>

### CODE REVIEW

**Comment by wborn** [comment]
Maybe we should just ignore all files in .settings? 

It's still possible to force add files ([source](https://stackoverflow.com/questions/24844782/how-to-override-a-rule-set-in-gitignore/24845089)) in case you do want to commit files that are ignored.

**Comment by davidgraeff** [comment]
I'm not sure if .settings can be of any use for future somethings. Let's be precautions and only ignore for now what we have identified as auto-generated.

**Comment by wborn** [comment]
At other projects using m2e I never add any Eclipse .settings files and I am very happy with it. :-) All that content gets regenerated when importing Maven project or when updating Maven projects (ALT+F5).

If you don't ignore them they bug you everytime in PRs and always show up as local changes after updates/imports.

**Comment by davidgraeff** [comment]
I know what you mean. That's the reason why I propose to ignore all `org.eclipse` files in `.settings`. But I'm not sure if there are open standards for text formatting etc or visual studio code tasks etc that we want to reside in there on purpose some time.

**Comment by wborn** [comment]
OK but it would make sense to delete all these files from the repo if they are ignored.

File: .gitignore [hunk 1]
```diff
@@ -21,18 +22,4 @@ distribution/openhabhome/userdata/mapdb/storage.mapdb
...
-addons/voice/org.openhab.voice.marytts/lib/jtok-core-1.9.1.jar
-addons/voice/org.openhab.voice.marytts/lib/marytts-lang-de-5.1-SNAPSHOT.jar
-addons/voice/org.openhab.voice.marytts/lib/marytts-lang-en-5.1-SNAPSHOT.jar
-addons/voice/org.openhab.voice.marytts/lib/mwdumper-1.16.jar
-addons/voice/org.openhab.voice.marytts/lib/opennlp-maxent-3.0.1-incubating.jar
-addons/voice/org.openhab.voice.marytts/lib/opennlp-tools-1.5.1-incubating.jar
-addons/voice/org.openhab.voice.marytts/lib/sgt-3.0.jar
-addons/voice/org.openhab.voice.marytts/lib/voice-bits1-hsmm-5.0-SNAPSHOT.jar
-addons/voice/org.openhab.voice.marytts/lib/voice-bits3-hsmm-5.0-SNAPSHOT.jar
-addons/voice/org.openhab.voice.marytts/lib/voice-cmu-slt-hsmm-5.0.jar
```
Comment (wborn) [inline .gitignore]: Why is this removed? The binding is still there and the JARs are still download during the build.

Comment (davidgraeff) [reply .gitignore]: I though marytts is already ported, because of that line in the same file: `bundles/io/org.openhab.io.multimedia.tts.marytts/lib/` but I know realize that the "io/" part renders this line ineffective anyway

**Comment by davidgraeff** [comment]
Creating a new PR

**Comment by Hilbrand** [comment]
@davidgraeff can you clean up your closed branches. It's now polluting the main openhab2-repository. Maybe also make those patches against your own fork. Now that you have write rights creating a patch online creates it against the main repository and not your own fork.

**Comment by davidgraeff** [comment]
Sorry for that. Some small patches are just done via the github interface, and github doesn't allow me to select my own fork as target. I'll remove those branches ASAP after merge or PR close.

IMO it is better to have those small patches and a bit of temporary branch pollution then not having those patches (because they are forgotten when back at the work station)
